### DISCUSSION TITLE
Automatic1111 extension

### DISCUSSION THREAD

**Discussion by yggdrasil75** [civitai/civitai#200] [Ideas]
I am sure most of those using these models use automatic1111, if not all. instead of having to go to civitai, it would be nice to have an extension that either embedded it in a tab on automatic, allowing simple downloading, or a way to "download to active model folder". if I am running automatic1111 at default port then civit can presumably attempt to send to an extension for automatic the file and have it downloaded straight to the proper folder, similar to how curseforge or nexusmods have "download with manager" that sends to the local mod manager.

**Comment by JustMaier** [comment]
We've been working on this. Stay tuned :)
